sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -391,6 +391,29 @@ class HiveDDLSuite
     }
   }
 
+  test("create view with mismatched schema") {
```

Comment:
Eventually, we will remove the functionality gap. At that moment, we need to clean the test suites and combine all the duplicate test cases.